alexandria/core/management/commands/migrate_document_title_and_description.py

```diff
@@ -0,0 +1,22 @@
+from django.core.management.base import BaseCommand
+from pghstore import loads
```

Comment:
Why do we need this additional dependency?

Reply:
I need this in the migration to parse the existing history content.